README.md

```diff
@@ -28,37 +28,78 @@ tools:
+![Level Zero](https://github.com/pocl/pocl/actions/workflows/build_level0.yml/badge.svg?event=push&branch=main)
+![Vulkan](https://github.com/pocl/pocl/actions/workflows/build_openasip_vulkan.yml/badge.svg?event=push&branch=main)
+![OpenASIP](https://github.com/pocl/pocl/actions/workflows/build_openasip_vulkan.yml/badge.svg?event=push&branch=main)
+![Remote](https://github.com/pocl/pocl/actions/workflows/build_remote.yml/badge.svg?event=push&branch=main)
+![Apple M1](https://github.com/pocl/pocl/actions/workflows/build_macos.yml/badge.svg?event=push&branch=main)
+
+### Support Matrix legend:
+
+:large_orange_diamond: Tested in CI extensively, including OpenCL-CTS tests
+
+:green_circle: : Tested in CI
+
+:yellow_circle: : Should work, but is untested
+
+:x: : Unsupported
+
+### Linux
+
+| CPU device  |     LLVM 14    |     LLVM 15    |     LLVM 16     |     LLVM 17    |     LLVM 18     |
+|:------------|:--------------:|:---------------:|:--------------:|:---------------:|:---------------:|
+| [x86-64](https://github.com/pocl/pocl/actions/workflows/build_linux_gh.yml) | :green_circle: | :green_circle:  | :green_circle: | :large_orange_diamond: | :large_orange_diamond: |
+| [ARM64](https://github.com/pocl/pocl/actions/workflows/build_arm64.yml) | :yellow_circle: | :yellow_circle: |:yellow_circle: | :yellow_circle: | :green_circle:  |
+| i686    | :yellow_circle: | :yellow_circle: | :yellow_circle: | :yellow_circle: | :yellow_circle: |
+| ARM32   | :yellow_circle: | :yellow_circle: | :yellow_circle: | :yellow_circle: | :yellow_circle: |
+| RISC-V  | :yellow_circle: | :yellow_circle: | :yellow_circle: | :yellow_circle: | :yellow_circle: |
+| PowerPC | :yellow_circle: | :yellow_circle: | :yellow_circle: | :yellow_circle: | :yellow_circle: |
+
+| GPU device  |     LLVM 17    |     LLVM 18     |
+|:------------|:--------------:|:---------------:|
+| [CUDA SM5.0](https://github.com/pocl/pocl/actions/workflows/build_cuda.yml) | :green_circle: | :green_circle: |
+| CUDA SM other than 5.0  | :yellow_circle: | :yellow_circle: |
+| [Level Zero](https://github.com/pocl/pocl/actions/workflows/build_level0.yml) | :green_circle: | :green_circle: |
+| [Vulkan](https://github.com/pocl/pocl/actions/workflows/build_openasip_vulkan.yml) | :green_circle: | :x: |
+
+| Special device |    LLVM 17    |     LLVM 18     |
+|:---------------|:-------------:|:---------------:|
+| [OpenASIP](https://github.com/pocl/pocl/actions/workflows/build_openasip_vulkan.yml) | :green_circle: | :x:            |
+| [Remote](https://github.com/pocl/pocl/actions/workflows/build_remote.yml) | :yellow_circle: | :green_circle:  |
+| Remote + RDMA  | :yellow_circle: | :green_circle:  |
+
+
+### Mac OS X
+
+| CPU device  |     LLVM 16    |     LLVM 17     |
+|:------------|:--------------:|:---------------:|
+| [Apple M1](https://github.com/pocl/pocl/actions/workflows/build_macos.yml) | :green_circle: | :green_circle:  |
+
+### Windows
+
+| CPU device  |     LLVM-MinGW 18    |  LLVM-MinGW 19  |
```

Comment:
MSVC?

Reply:
I have no idea about MSVC ? when Henry finishes his work he can add it to the table.